autocannon.js

```diff
@@ -139,9 +151,37 @@ function start (argv) {
     return
   }
 
+  if (argv.onPort) {
+    const proc = nitm(
+      ['-r', require.resolve('./lib/detectPort')],
```

Comment:
You should use NODE_PATH instead, as this will fail on windows with usernames with spaces in them (which lots of people have there).

Move detectPort.js to  another folder, fx `./injects` and set `process.env.NODE_PATH = path.join(__dirname, '/injects')` and use `['-r', 'detectPort']` to work around that.

Reply:
I took it from https://github.com/davidmarkclements/0x/blob/3611fc7f7c04ee741daeda33c972bb26cd7bdd0b/platform/v8.js#L34, so we should also change it there then. Does windows not support quoted arguments? :scream: